samples/bluetooth/direct_test_mode/src/fem/fem.c

```diff
@@ -276,6 +276,13 @@ int8_t fem_tx_output_power_prepare(int8_t power, int8_t *radio_tx_power, uint16_
 	return output_power;
```

Comment:
Is using `false` in https://github.com/nrfconnect/sdk-nrf/blob/8c5af24cb14f2cea9a0b566af677ffee22a546f0/samples/bluetooth/direct_test_mode/src/fem/fem.c#L265 is correct from DTM-s and BT core specifications perspective?

Reply:
We check the nearest possible value before this function call so I suppose that this function returns the same output power for ceiling or floor because we pass the supported value to this function. Or maybe I missed something.